src/components/SearchTracePage/index.js

```diff
@@ -18,29 +18,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import _values from 'lodash/values';
```

Comment:
Ok cool, im open to the lodash function style of _values as an alias for _.values. It would be sweet if we could use this babel plugin but then we'd have to eject. https://github.com/lodash/babel-plugin-lodash